### PR TITLE
Update activity and health events data.

### PR DESCRIPTION
**Pull request by RazumeikoAlex**

### CODE REVIEW

File: source/includes/live/_webhooks.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Your endpoint implementation must retrieve the `SubscribeURL` value from the sub
 
 ## Processing webhooks
 
-Once you have confirmed the registration URL above, you will see an array of location and activity change events in POST body. There might be multiple events if they were pushed in short period of time (1 second or less).
+Once you have confirmed the registration URL above, you will see an array of location, health and activity change events in POST body. Activity and health events will contains snapshot last device status like last location, health, activity. There might be multiple events if they were pushed in short period of time (1 second or less).
```
Comment (agraebe) [inline source/includes/live/_webhooks.md]: "Activity and health events contain a snapshot of the last device status (location, health, activity)"

File: source/includes/live/_webhooks.md [hunk 2]
```diff
@@ -69,6 +70,7 @@ Once you have confirmed the registration URL above, you will see an array of loc
 |recorded_at                 |string       | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date when the location was recorded. Not used in summary type
 |type                        |string       | Type of webhook data. Could be either `location`, `activity`, `health`, `summary`
 |data                        |object       | Either `location`, `activity`, `health` or `summary` data, see details below.
+|device_movement_status      |object       | Last known device status. Available in `activity` and `health` events.
```
Comment (agraebe) [inline source/includes/live/_webhooks.md]: What's the format of the object? If the format is the same as some other object, we could communicate that easy

**Comment by RazumeikoAlex** [comment]
@agraebe added changes.

**Comment by agraebe** [comment]
@RazumeikoAlex I added your changes to my commit due to conflicts (I modified the folder structure for the SDK references)

**Pull request closed without merging**